products/add-data-to-grid-datastores/datastore-faq.md

```diff
@@ -20,6 +20,10 @@ When you create a datastore, you certify that you have the rights to that data.
 
 Grid does not charge for storing data. However, there is a limit of 25 datastores.
 
+## Does Grid have a limit on DataStore size?
+
+Yes Grid DataStore max size is 1.5 TB. In addition, for security reasons we only allow you to upload data for 24 hours. If your data is too big to upload in 24 hours you will need to manually resume the upload every 24 hours.
```

Comment:
Should we rephrase it to something like this to avoid confusion about what's "manually resume" means?

```
we only allow you to upload data for 24 hours continuously. If your data is too big to upload in 24 hours, you session will break. But initiating `grid datastore create <args...>` again will resume the upload instead of starting from scratch again 
```